.github/ISSUE_TEMPLATE/Feature_Request.md

```diff
@@ -0,0 +1,47 @@
+---
+name: 🚀 Feature Request
+about: I have a suggestion (might want to implement it myself 🙂)!
+labels: enhancement
+additional label: new_module/new_plugin/documentation/feature
+---
+
+<!--- Please keep this note for the community --->
+
+### Community Note
+
+* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
+* If you are interested in working on this issue or have submitted a pull request, please leave a comment
+
+<!--- Thank you for keeping this note for the community --->
+
+### Description
+
+<!--- Please leave a helpful description of the feature request here. --->
+
+* xxxx
+
+### New or Affected Module(s):
+
+<!--- Please list the new or affected module(s). --->
+
+* mso_XXXXX + fv:XXXX
+
+### MSO version and MSO Platform
+
+* V x.x.x and on-prem/cloud-aws/cloud-azure/all.
```

Comment:
and Docker/OVA/MSO (to replace the on-prem....)

Also add another section for:
APIC version and APIC Platform for Site Level Resources
V x.x.x and on-prem/cloud-aws/cloud-azure/all.